content/04.results.md

```diff
@@ -14,15 +14,16 @@ Panel B shows the types and amounts of articles that we have used for analyses.
 
 ### Creation of an Annotated News Dataset
 
-We have analyzed the text and citations of 22,094 news-related articles hosted on "www.nature.com".
-The articles span 15 years from 2005 to 2020, and seven article types: Career Column, Career Feature, News, News and Views, News Feature, Technology Feature, and Toolbox.
+We have analyzed the text and citations of 16,080 news-related articles hosted on "www.nature.com".
+The articles span 15 years from 2005 to 2020, and five article types: Career Feature, News, News Feature, Technology Feature, and Toolbox.
```

Comment:
Then here you could say that our primary focus is on 16,080 articles of types authored by journalists: Career Feature, News, News Feature, Technology Feature, and Toolbox.

Reply:
yeah, I think this makes sense changed in https://github.com/greenelab/nature_news_manuscript/pull/3/commits/27be112764daab13da1f30c1027bf6d3dd3e9327